.github/workflows/container-image.yml

```diff
@@ -84,6 +96,16 @@ jobs:
           outputs: type=docker,dest=/tmp/kubewarden-controller-image-${{ env.TAG_NAME }}.tar
           tags: |
             ghcr.io/${{github.repository_owner}}/kubewarden-controller:${{ env.TAG_NAME }}
+      -
+        name: Create SBOM file
+        shell: bash
+        run: |
+          bom generate -n https://kubewarden.io/kubewarden.spdx -o kubewarden-controller.spdx .
+      -
+        name: Attach SBOM file in the container image
+        shell: bash
+        run: |
+          cosign attach sbom --sbom kubewarden-controller.spdx ghcr.io/${{github.repository_owner}}/kubewarden-controller:${{ env.TAG_NAME }}
```

Comment:
I think this is just attaching the sbom, but it does not sign it. This is the output I see:
```
cosign attach sbom --sbom kubewarden-controller.spdx raulcabm/sigstore:cosign
WARNING: Attaching SBOMs this way does not sign them. If you want to sign them, use 'cosign attest -predicate kubewarden-controller.spdx -key <key path>' or 'cosign sign -key <key path> <sbom image>'.
```

Reply:
Can we use the digest instead of the tag? This is how we do it for signing https://github.com/kubewarden/kubewarden-controller/blob/main/.github/workflows/container-build.yml#L36

Reply:
Oh, for some reason the signing step is not committed. I did it already. Let me check what happened

Reply:
I've updated the PR pushing the missing steps. I'll update the PR to use digest next.